libs/coin-modules/coin-evm/src/synchronization.ts

```diff
@@ -67,7 +67,7 @@ export const getAccountShape: GetAccountShape<Account> = async (infos, { blackli
           Math.max(latestSyncedHeight - SAFE_REORG_THRESHOLD, 0),
           blockHeight,
         );
-      } catch (e) {
+      } catch (e) /* istanbul ignore next: just logs */ {
```

Comment:
why istanbul comments everywhere ?

Reply:
Because that's parts of the code that we don't want/need to test, so it's unnecessary in the code coverage to have them as uncovered and potentially showing files as yellow or red as is if it was problematic.
TLDR: Nicer statistics